packages/eslint-plugin-wpcalypso/lib/rules/test/i18n-unlocalized-url.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file Disallow using unlocalized URL strings
+ * @author Automattic
+ * @copyright 2023 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const RuleTester = require( 'eslint' ).RuleTester;
+const rule = require( '../i18n-unlocalized-url' );
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const localizableUrls = [
+	'https://apps.wordpress.com/',
```

Comment:
Care to add a few more fixtures to cover all scenarios? 

Some examples I can think of are:

* https://apps.wordpress.com/mobile/
* https://automattic.com/cookies (without the trailing slash)
* all other URLs that are intended to have a trailing slash, but will work without one too due to the [WordPress canonical redirects](https://developer.wordpress.org/reference/functions/redirect_canonical/).

Reply:
Thanks for the suggestion. Added more fixtures in https://github.com/Automattic/wp-calypso/pull/78480/commits/2489f7a3be20ad3d3c8533b028a257e0972d013c which should cover pretty much all URLs that we match.